gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -185,6 +186,17 @@ def generate_allele_data(ht: hl.Table) -> hl.Table:
 def generate_ac(mt: hl.MatrixTable) -> hl.Table:
     """
     Creates Table with QC samples, QC samples removing children and release samples raw and adj ACs.
```

Comment:
I had to re-read this comment line a few times over and think it could be clarified by simply removing the line and letting stand the new docstring comments below (i.e., "Returns table containing the following annotations:")